tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -2302,10 +2301,9 @@ <h2>Show similarity to selected datapoint</h2>
                                 slices achieve equal opportunity.
                               </div>
                               <div>
-                                Equal opportunity means that among those
-                                predicted as positive classifications, there is
-                                a similar percentage of correct predictions in
-                                each slice.
+                                Equal opportunity means that a similar fraction
+                                of the real positives is recalled for each slice
+                                (i.e., <i>sensitivity</i> is the same).
```

Comment:
I think this definition is confusing in its use of "real positives" and "recalled" as we want this to be clear even to novices. Let's keep it as close to the initial wording as possible.

Perhaps "Equal opportunity means that among those datapoints with the positive ground truth label, there is a similar percentage of correct predictions in each slice".

Reply:
Sounds good!